bofire/data_models/features/tasks.py

```diff
@@ -0,0 +1,32 @@
+from typing import List, Literal
+
+import numpy as np
+from pydantic import model_validator, validator
+
+from bofire.data_models.features.api import DiscreteInput
+
+
+class TaskInput(DiscreteInput):
+    type: Literal["TaskInput"] = "TaskInput"
+    n_tasks: int
```

Comment:
`n_tasks` is not necessary if you base it on the CategoricalInput as it would be just the number of categories.